### PR TITLE
feat: Mail account delegation

### PR DESCRIPTION
**Pull request by ChristophWurst**
## ☑️ Resolves

* Fix https://github.com/nextcloud/documentation/issues/10017
* Close https://github.com/nextcloud/mail/issues/7723

### Admin docs

Bildschirmfoto vom 2023-05-05 12-08-00

### User docs
Bildschirmfoto vom 2023-05-05 11-59-27

### CODE REVIEW

File: user_manual/groupware/mail.rst [hunk 1]
```diff
@@ -50,6 +50,16 @@ Your account settings such as:
 
 Can be found in the action menu of a mail account. There you can edit, add or remove settings depending on your need.
 
+Account delegation
+~~~~~~~~~~~~~~~~~~
+
+The app allows account delegation so that one user can send emails from the address of another.
+
+1) The delegation has to be configured on the mail server by an admin
+2) Add the other account as alias of your own
```
Comment (miaulalala) [inline user_manual/groupware/mail.rst:59]:
```suggestion
2) Add the other email address as an alias for your own email account.
```

**Review by miaulalala — CHANGES_REQUESTED** [review]

**Review by miaulalala — APPROVED** [review]

**Comment by ChristophWurst** [comment]
There are no mail docs in 25

**Pull request merged**